yarn-project/archiver/src/archiver/memory_archiver_store/memory_archiver_store.ts

```diff
@@ -288,7 +288,7 @@ export class MemoryArchiverStore implements ArchiverDataStore {
         const currentLogs = this.taggedLogs.get(tag.toString()) || [];
         this.taggedLogs.set(tag.toString(), [
           ...currentLogs,
-          new TxScopedL2Log(txHash, dataStartIndexForTx, block.number, /* isFromPublic */ false, log.toBuffer()),
+          new TxScopedL2Log(txHash, dataStartIndexForTx, block.number, log),
```

Comment:
TxScopedL2Log now contains the PublicLog, PrivateLog type info so isFromPublic param is unnecessary now.